### PR TITLE
Add podSecurityStandards.enforced

### PR DESCRIPTION
**Pull request by mnitchev**
### What does this PR do?

`Sets policy.giantswarm.io/psp-status: disabled` on the Cluster CR. This is needed for upgrading to kubernetes 1.25

### What is the effect of this change to users?

None, this is just necessary for upgrading to 1.25

### Any background context you can provide?

Towards https://github.com/giantswarm/roadmap/issues/2925

### Do the docs need to be updated?
no

### Should this change be mentioned in the release notes?

- [x] CHANGELOG.md has been updated (if it exists)

### CODE REVIEW

**Comment by nprokopic** [comment]
@mnitchev we'll need this only during the v1.24 -> v1.25 transition phase, right?

**Comment by mnitchev** [comment]
Yeah that's what this is for. I'm currently testing it in cluster-aws here https://github.com/giantswarm/cluster-aws/pull/481

**Review by T-Kukawka — APPROVED** [review]

File: helm/cluster/templates/clusterapi/controlplane/_helpers_clusterconfiguration_apiserver.tpl [hunk 1]
```diff
@@ -135,6 +135,9 @@ api-audiences-example.giantswarm.io
 
 {{- define "cluster.internal.controlPlane.kubeadm.clusterConfiguration.apiServer.featureGates" }}
 {{- $featureGates := list -}}
+{{- if semverCompare "<1.25-0" .Values.providerIntegration.kubernetesVersion -}}
+{{- $featureGates = append $featureGates "TTLAfterFinished=true" -}}
```
[the remark this answers is not in the document]
Comment (mnitchev) [reply helm/cluster/templates/clusterapi/controlplane/_helpers_clusterconfiguration_apiserver.tpl:139]: I was just following the cluster-azure PR. I'll get rid of it :D

[the remark this answers is not in the document]
Comment (mnitchev) [reply helm/cluster/templates/clusterapi/controlplane/_helpers_clusterconfiguration_apiserver.tpl]: Done.

**Pull request merged**